core/src/main/resources/error/error-classes.json

```diff
@@ -616,6 +616,11 @@
     ],
     "sqlState" : "42000"
   },
+  "INVALID_EMPTY_LOCATION" : {
+    "message" : [
+      "A specified location cannot be empty."
```

Comment:
This still confuses me. The location path doesn't contain any data/files, or it is an empty string, or it is null. Looking at:
```scala
      if (StringUtils.isEmpty(location)) {
        throw QueryExecutionErrors.unsupportedEmptyLocationError()
      }
```
and at `StringUtils.isEmpty`:
```scala
    public static boolean isEmpty(final CharSequence cs) {
        return cs == null || cs.length() == 0;
    }
```
the error is about the path string, not data.

@itholic Could you make the error more precise, and rename `QueryExecutionErrors.unsupportedEmptyLocationError`, please.